app/assets/stylesheets/config/_base.css

```diff
@@ -0,0 +1,48 @@
+:root {
+  --script-size-s: 3.25rem;
+  --script-size-m: 4.5rem;
+  --script-size-l: 5.5rem;
+  --font-size-xs: 1.6rem;
+  --font-size-s: 2rem;
+  --font-size-m: 2.5rem;
+  --font-size-l: 4rem;
+  --font-size-xl: 6.5rem;
+}
+
+*, *::before, *::after {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+}
+
+/* body *, body *::before, body *::after {
+  border: 0.5px solid blue;
+} */
+
+html {
+    /*
+  This allows us to use readable rems.
+  - 24px becomes 2.4rem
+  - 10px becomes 1rem
+  - etc
+   */
+  font-size: 62.5%;
+}
+
+html, body { height: 100% }
```

Comment:
**question:** Why setting height?

Reply:
It was for something I tried before and forgot to delete 😬